project/currency_comparison/__tests__/currency_comparison.test.js

```diff
@@ -0,0 +1,50 @@
+const CurrencyComparison = require('../currency_comparison')
+const axios = require('axios')
+const { test } = require('@jest/globals')
+jest.mock('axios')
+
+const testSalary = new CurrencyComparison(50000)
+
+describe("Outputs the number of work hours needed to obtain an inputted salary in USD, earning foreign currency", () => {
+    test("Calculates number of hours per week needed to reach yearly salary goal", () => { })
+    test("Converts foreign currency(Ex. CAD) to USD", async() => {
+        //arrange
+        const currencyCode = 'CAD'
+        const expectedValue = .82
+        //act
+        const actualValue = await testSalary.currencyConversion(currencyCode) 
+        //assertions
+        expect(actualValue).toBe(expectedValue)
+    })
+    test("Converts foreign currency(Ex. EUR) to USD", async() => {
+        //arrange
+        const currencyCode = 'EUR'
+        const expectedValue = 1.21
+        //act
+        const actualValue = await testSalary.currencyConversion(currencyCode) 
+        //assertions
+        expect(actualValue).toBe(expectedValue)
+    })
```

Comment:
1. These two tests can be combined into one. A single test can have multiple expect assertions.
2. `testSalary.currencyConversion` isn't an asynchronous function so this shouldn't use `async` and `await`?